src/App.tsx

```diff
@@ -10,9 +13,15 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/people">
+              <Route index element={<PeoplePage />} />
```

Comment:
You can combine these two like

```js
<Route path=":personSlug?" element={<PeoplePage />} />
```

the question mark does it all // but please don't worry about it, I only wanted to say